### PR TITLE
Added Gretchen’s FAQ on aggregation tools

### PR DESCRIPTION
**Pull request by hannahfrost**

### CODE REVIEW

File: _data/faqs.yaml [hunk 1]
```diff
@@ -23,6 +23,10 @@
...
 - question: >-
+    Will Hydra-in-a-Box include metadata aggregation tools?
+  answer: >-
+    The short answer is no, however, the Hydra-in-a-Box grant project includes further development of DPLA’s ingestion system, Heiðrún, to create a more robust suite of aggregation tools to be used by DPLA hubs and others engaged in aggregating, remediating, and enriching metadata from multiple sources. While this product will not be tightly coupled with Hydra-in-a-Box, the two will be complementary and may share some componentry. And while it will be easy to use Hydra-in-a-box and the aggregation tools together, the two will be fully featured and operate independently. In other words, Hydra-in-a-Box will not be required to run "Aggregator-in-a-Box" and vice versa. More information about Heiðrún is <a href="https://digitalpubliclibraryofamerica.atlassian.net/wiki/display/TECH/Heidrun">available on the DPLA wiki</a>.
```
Comment (mjgiarlo) [inline _data/faqs.yaml]: @hannahfrost @anarchivist Only change I might make is to capitalize "box" in `...easy to use Hydra-in-a-box ...`. Do we care about that or meh?

Comment (anarchivist) [inline _data/faqs.yaml]: Probably worth it to fix, in my opinion...

**Comment by hannahfrost** [comment]
Good catch, and it is now fixed.

**Comment by mjgiarlo** [comment]
:+1: thanks!

**Pull request merged**